src/Microsoft.ML.Data/Transforms/TransformsCatalog.cs

```diff
@@ -37,6 +37,11 @@ public sealed class TransformsCatalog
         /// </summary>
         public FeatureSelectionTransforms FeatureSelection { get; }
 
+        /// <summary>
+        /// List of operations for using TensorFlow model.
+        /// </summary>
+        public TensorFlowTransforms TensorFlow { get; }
```

Comment:
>TensorFlowTransforms [](start = 15, length = 20)

Please do not do this. Otherwise we will have an empty property unless someone imports the nuget, which is confusing and undesirable.

Please follow instead the pattern that we see in image processing. You'll note that we do not have an empty image processing nuget. Rather they are added to this catalog.

This is defensible since we can take someone directly importing a nuget as a strong signal that they want to actually use those transforms.